app/controllers/state_file/questions/unemployment_controller.rb

```diff
@@ -1,9 +1,9 @@
 module StateFile
   module Questions
     class UnemploymentController < AuthenticatedQuestionsController
-      include ReturnToReviewConcern
       include OtherOptionsLinksConcern
-      before_action :load_faq_link, only: [:new, :edit]
+      include ReturnToReviewConcern
+      before_action :load_links, only: [:new, :edit]
```

Comment:
[this](https://stackoverflow.com/questions/68815705/does-before-action-in-class-override-actions-defined-in-concern) is what i'm doing here